scripts/bump.py

```diff
@@ -0,0 +1,181 @@
+"""Script for bumping core dependencies."""
+
+import typing as t
+from pathlib import Path
+
+import click
+import requests
+from aea.cli.utils.click_utils import PyPiDependency
+from aea.configurations.constants import PACKAGES, PACKAGE_TYPE_TO_CONFIG_FILE
+from aea.configurations.data_types import Dependency
+from aea.helpers.logging import setup_logger
+from aea.helpers.yaml_utils import yaml_dump_all, yaml_load_all
+from aea.package_manager.v1 import PackageManagerV1
+
+from autonomy.cli.helpers.ipfs_hash import load_configuration
+
+
+BUMP_BRANCH = "chore/bump"
+PIPFILE = Path.cwd() / "Pipfile"
+PYPROJECT_TOML = Path.cwd() / "pyproject.toml"
+TOX_INI = Path.cwd() / "tox.ini"
+TAGS_URL = "https://api.github.com/repos/{repo}/tags"
+OPEN_AEA_REPO = "valory-xyz/open-aea"
+OPEN_AUTONOMY_REPO = "valory-xyz/open-autonomy"
+
+_repo_version_cache = {}
+
+_logger = setup_logger("bump")
+
+
+def get_latest_tag(repo: str) -> str:
+    """Fetch latest git tag."""
+    response = requests.get(url=TAGS_URL.format(repo=repo))
+    if response.status_code != 200:
+        raise ValueError(
+            f"Fetching tags from `{repo}` failed with message '"
+            + response.json()["message"]
+            + "'"
+        )
+    latest_tag_data, *_ = response.json()
+    _repo_version_cache[repo] = latest_tag_data["name"]
+    return _repo_version_cache[repo]
+
+
+def get_open_aea_dependencies() -> t.Dict[str, str]:
+    """Get open-aea dependencies."""
+    version = get_latest_tag(OPEN_AEA_REPO).replace("v", "")
```

Comment:
Works most of the time. But what if they're different...? 

More robust approach: scan the repo to be bumped for `open-aea-*` deps and then search for the correct new version in OPEN_AEA_REPO